src/index.html

```diff
@@ -0,0 +1,273 @@
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+          <div class="speaker">
+            <img
+              class="speaker__image"
+              src="./assets/images/speaker-1.jpg"
+              alt="Speaker 1"
+            />
+            <div class="speaker__body">
+              <div class="speaker__name">Yochai Benkler</div>
+              <p class="speaker__occupation">
+                Berkman Professor of Entrepreneurial Legal Studies at Harvard
+                Law School
+              </p>
+              <p class="speaker__about">
+                Benkler studies commons-based peer production, and published his
+                senminal book The Wealth of Networks in 2006
+              </p>
+            </div>
+          </div>
+        </div>
+        <button class="section__more-btn" type="button">
+          <span>More</span>
+          <img src="./assets/icons/chevron-down.svg" alt="Chevron down icon" />
```

Comment:
- [ ] Kindly make sure that when more button is clicked in the mobile version the remaining speakers are displayed.